elasticdl/python/worker/ps_client.py

```diff
@@ -92,3 +100,102 @@ def push_dense_parameters(self, parameters, ps_id, version):
         for p in parameters:
             serialize_ndarray(p.values, model.dense_parameters[p.name])
         self.ps_stubs[ps_id].push_model(model)
+
+    def push_gradients(
+        self, grads, edl_grads, learning_rate, model_versions,
+    ):
+        """
+        Push gradients to PS. There three kinds of gradients:
+         - gradients
+         - sparse gradients of ElasticDL embedding layers
```

Comment:
The docstring only displays 2 kinds of gradients.

Reply:
Done